dev-server/static/scripts/pdfjs-2/LICENSE

```diff
@@ -0,0 +1,177 @@
+
```

Comment:
The `pdfjs-2` name was originally used in Via because we were doing the upgrade from PDF.js v1 to v2. I would suggest to just use `pdfjs` here.